### PR TITLE
Added LIA and ERR status byte read properties to the SRS Sr830 driver

### PR DESCRIPTION
**Pull request by samcondon4**
Small PR. Just adding two additional properties to read status bytes from the sr830. The queries for these status bytes don't follow SCPI so new properties need to be created.

### CODE REVIEW

File: pymeasure/instruments/srs/sr830.py [hunk 1]
```diff
@@ -89,6 +89,36 @@ class SR830(Instrument):
...
+           bit 2: RAM error
+           bit 3: unused
+           bit 4: ROM error
+           bit 5: GPIB error
+           bit 6: DSP error
+           bit 7: Math error
+           """,
+        get_process=lambda s: bin(int(s))[2:],
+    )
+
```
Comment (msmttchr) [inline pymeasure/instruments/srs/sr830.py:121]: In both cases you return a bit mask which identifies the error conditions.
When users see the mask, they need to go back in the manual to decode the mask.
Now, python has some more descriptive data type for such cases.
For example, you could try to use `IntFlag` or `Flag` as suggested by @bilderbuchi
sometime ago in a similar situation.
In such a way users can directly see the meaning of the flags.

**Review by msmttchr — CHANGES_REQUESTED** [review]
Thanks for your PR. I suggested some changes to ease readability of results for users.

File: pymeasure/instruments/srs/sr830.py (hunk 1 above)
Comment (samcondon4) [reply pymeasure/instruments/srs/sr830.py:148]: Good point. I had not used the IntFlag type before this. Just implemented this change.

**Review by msmttchr — APPROVED** [review]
Thanks for your work, looks good to me.

**Pull request merged**